easy_study/lib/model/Subject.dart

```diff
@@ -1,26 +1,23 @@
 import 'dart:ui';
 
+import 'package:easy_study/Database/DBHelper.dart';
 import 'package:easy_study/model/Priority.dart';
 import 'package:easy_study/model/Type.dart';
 
 class Subject {
-
   String _title, _room, _description;
   int _id, _timeSpent = 0, _hoursWeek;
   Type _type;
   Priority _priority;
+  String _startTimetracking;
```

Comment:
It's not completely sure, what u mean with the name.  Also, add documentation to written code.
```suggestion
  String _startedTimetrackingAt;
```
If u change the name, u have to change everything who is using this. Getter setter and users.